flow-typed/npm/mocha_v5.x.x.js

```diff
@@ -0,0 +1,222 @@
+// flow-typed signature: ae94222964881f91f3333824878bbb2f
```

Comment:
Should this have been committed to the repo?

Reply:
yes, unfortunately there isn't a tool like `@types` for flow.